std_msgs/String commandString
---
bool commandExecuted